docs/platform/release-notes/tizen-2-4.md

```diff
@@ -850,7 +844,6 @@ The Tizen 2.4 release provides developers with the Tizen kernel, device drivers,
   - When you import some projects generated from the Tizen 2.3 Rev2 SDK, the importing can take some time.
 - Emulator
   - To use the Tizen Emulator, you must install an Intel VTx supported by the CPU, and the latest version of the graphic card driver which the vendor provides. Check the prerequisites for the Tizen Emulator from:
-    - [https://developer.tizen.org/downloads/sdk/installing-sdk/prerequisites-tizen-sdk](https://developer.tizen.org/downloads/sdk/installing-sdk/prerequisites-tizen-sdk).
```

Comment:
How about removing "Check the prerequisites for the Tizen Emulator from:" of above line also?